File: dayFour/afterBreak/java/TaskSeven.java
import java.util.*;


public class TaskSeven {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int evenScoreSum = 0;
	double evenScoreAverage = 0;
	int evenScoreCounter = 0;
	
	System.out.println("Enter the scores below: ");
	
	
	for(int counter= 0; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    if(score % 2 == 0)
	      {
	        evenScoreSum += score;
	        evenScoreCounter++;
	      } else { }
	  }
	  
	  evenScoreAverage = evenScoreSum/evenScoreCounter;
	System.out.println("The sum of even scores is: " + evenScoreSum);
	}
}

File: dayFour/afterBreak/python/taskSeven.py
even_sum_total = 0
even_score_average = 0
even_counter = 0
even_counter = int(even_counter)

print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    if score % 2 == 0:
        even_sum_total = even_sum_total + score
        even_counter += 1

even_score_average = even_sum_total/even_counter
print("The average of even scores is:", even_score_verage)
print("The sum of even scores is:", even_sum_total)
